Sources/SwiftSyntaxBuilder/SourceFileConvenienceInitializers.swift

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+extension SourceFile {
+    /// A convenience initializer that allows:
+    ///  - Initializing syntax collections using result builders
+    ///  - Initializing tokens without default text using strings
+    public init(
+      @CodeBlockItemListBuilder statementsBuilder: () -> ExpressibleAsCodeBlockItemList = { CodeBlockItemList([]) }
```

Comment:
I don’t think having a default value here really makes sense. You very rarely want to generate an empty source file 😉